kubectl/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM amazon/aws-cli:latest
+ARG KUBECTL_VERSION=v1.18.5
+RUN curl -sLo /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl \
+  && chmod +x /usr/local/bin/kubectl
+
```

Comment:
Can you also add `jq`, we need this for the rolling-upgrades? See original kops image.